syft/dependency_check.py

```diff
@@ -1,10 +1,33 @@
+from __future__ import absolute_import
+from distutils.version import LooseVersion
+import importlib
 import logging
 
 logger = logging.getLogger(__name__)
 
 try:
-    import tf_encrypted as tfe
+    import tensorflow
+
+    if LooseVersion(tensorflow.__version__) < LooseVersion("2.0.0"):
+        raise ImportError()
+    pstf_spec = importlib.util.find_spec("syft_tensorflow")
+    tensorflow_available = pstf_spec is not None
+except ImportError:
+    tensorflow_available = False
+
+
+try:
+    tfe_spec = importlib.util.find_spec("tf_encrypted")
```

Comment:
Why do we use this method here but actually import tf

Reply:
We're actually only checking to see if a package is available here. Actually importing the package is unnecessary to complete that task, so it should be handled elsewhere if needed.